processing/src/main/java/io/druid/query/groupby/epinephelinae/RowBasedGrouperHelper.java

```diff
@@ -1677,30 +1691,67 @@ public void getFromByteBuffer(ByteBuffer buffer, int initialOffset, int dimValId
       @Override
       public int compare(ByteBuffer lhsBuffer, ByteBuffer rhsBuffer, int lhsPosition, int rhsPosition)
       {
-        return Double.compare(
-            lhsBuffer.getDouble(lhsPosition + keyBufferPosition),
-            rhsBuffer.getDouble(rhsPosition + keyBufferPosition)
-        );
+        return bufferComparator.compare(lhsBuffer, rhsBuffer, lhsPosition, rhsPosition);
       }
     }
+  }
 
-    private class LimitPushDownDoubleRowBasedKeySerdeHelper extends DoubleRowBasedKeySerdeHelper
-    {
-      final StringComparator cmp;
-
-      public LimitPushDownDoubleRowBasedKeySerdeHelper(int keyBufferPosition, StringComparator cmp)
-      {
-        super(keyBufferPosition);
-        this.cmp = cmp;
+  private static int compareDimsInBuffersForNullFudgeTimestamp(
+      List<RowBasedKeySerdeHelper> serdeHelpers,
+      int dimCount,
```

Comment:
If it's `serdeHelpers.size()`, why need to pass?

Reply:
I just moved this method as it was. Removed now.